adops/setting-up-prebid-video-in-freewheel.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: page_v2
+title: Setting up Prebid Video in Freewheel
+head_title: Setting up Prebid Video in Freewheel
+description: Setting up Prebid Video in Freewheel
+pid: 3
+hide: false
+top_nav_section: adops
+nav_section: tutorials
+sidebarType: 3
+---
+
+# Freewheel Guide for Premium Long-Form Video
```

Comment:
when I navigate to this page, it's not opening in the left nav

Reply:
This was not initially working for me locally but it now is.